onnx/hub.py

```diff
@@ -272,3 +273,87 @@ def load(
             )
 
     return onnx.load(cast(IO[bytes], BytesIO(model_bytes)))
+
+
+def download_test_data(
```

Comment:
So the returned directory will have both the model file and test data? If that is the case, perhaps this function name should be something like `download_model_with_test_data` to prevent confusion.

Reply:
Fixed in https://github.com/onnx/onnx/pull/4741/commits/b4d68a6c640c6447c9db36f336b9663db901d307.